x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/session/EsqlCCSUtilsTests.java

```diff
@@ -766,35 +762,6 @@ private void assertLicenseCheckFails(
         assertThat(e.status(), equalTo(RestStatus.BAD_REQUEST));
     }
 
-    public void testShouldIgnoreRuntimeError() {
```

Comment:
This test is pretty much pointless now as shouldIgnoreRuntimeError doesn't have any logic left in it.